src/components/Filters/index.js

```diff
@@ -311,12 +311,14 @@ class Filters extends React.Component {
                       name => <option key={name}>{name}</option>
                     )}
                   </Select>
-                  <FilterInput
-                    schema={this.props.schema[this.state.edit.name]}
-                    value={this.state.edit.value}
-                    onChange={value => this.handleEditChange(value, 'value')}
-                    type={inputModels[this.state.edit.name].type}
-                  />
+                  {inputModels[this.state.edit.name].type !== 'Boolean' && (
```

Comment:
I guess that having an `shouldShowInput()` method might be an overkill.

Reply:
AFAIK the boolean value is the only instance of this behaviour